### ISSUE TITLE
anchor size

### ISSUE THREAD

**Issue by ps793** [ringringyi/DOTA_YOLOv2#1]
I may have a question related to your anchor size. How do you modify the anchor size in your cfg files? I found not all images after splitting in training are 1024*1024 so that it may need to resize images when training yolo. will it have effect on anchor sizes?

**Comment by dingjiansw101** [comment]
I don't think it will affect the anchor size, but the aspect of the target in the image has changed. I think this will have a bad effect on the results, but we have not optimized for this situation.

**Comment by ps793** [comment]
I still have 2 question.
1) I think in your implementation of yolo-transform, you will also generate yolo txt files for those images which do not have object. Will it give error when you train the model? 
2) why did you divide 1024 for all the images? some images are not 1024 *1024. For the relative ratio of width and height, I think it should be divided by the size of images rather than 1024. 

Thank you!

**Comment by dingjiansw101** [comment]
For 1. You can think that it gives more negative samples to train the network. But I am not quite sure if it has positive or negative effects.
For 2. I think you are right. It may be a bug. You can try to fix it and see if there is any performance boost.
